.changeset/techdocs-quick-owls-smile.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-techdocs-addons-contrib': minor
```

Comment:
Probably. A bit unsure what goes into all the different ones. 😅

Reply:
To avoid confusion with the addon framework I think we can keep the `contrib` suffix, so we can go with `@backstage/plugin-techdocs-module-addons-contrib` 😅

Reply:
Yes I agree, I'm still learning. I'll loop in @Rugvip too here to see if it make sense!

Reply:
So this is a package where you intend to collect any kind of generic contributed addons right? In that case I think `@backstage/plugin-techdocs-module-addons-contrib` makes perfect sense 👍 

Only other thought is whether the split into packages should be more granular, with the risk of this one getting a bit big. Tbh that's something we can tweak in the future by splitting things out and re-exporting from here so wouldn't sweat it too much